source/Templates/NewStep/.template.config/template.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Octopus Deploy",
+  "classifications": ["Sashimi"],
+  "description": "A get started template for Sashimi projects",
+  "name": "Sashimi Handler Application",
+  "defaultName": "MySashimi",
+  "identity": "Octopus.Sashimi",
+  "groupIdentity": "Octopus.Sashimi",
+  "tags": {
+    "language": "C#",
+    "type": "project"
+  },
+  "shortName": "sashimi",
+  "sourceName": "NamingIsHard",
```

Comment:
Consider changing `NamingIsHard` to `ProjectNamePlaceHolder` or `ProjectNameToken`